firebase-config/test-app/src/androidTest/kotlin/com/google/firebase/testing/sessions/FirebaseConfigTest.kt

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
I know on the RC side Kotlin is basically just a wrapper of Java but I was wondering if we would need one app in Java and another in Kotlin so that we test both code paths

Reply:
Sounds good